x/posts/abci.go

```diff
@@ -0,0 +1,25 @@
+package posts
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/desmos-labs/desmos/x/posts/internal/types"
+)
+
+// EndBlocker called upon each block end to close expired polls
+func EndBlocker(ctx sdk.Context, keeper Keeper) {
+
+	posts := keeper.GetPosts(ctx)
+
+	for _, post := range posts {
```

Comment:
Instead of using a `for` loop on the entire list of posts, I would prefer we use an `Iterator` as this way the avoid going through all the posts two times (one to get the list and the other one to close the posts). 

Also, we should open up an issue reminding ourselves to improve this process by iterating only the posts that are still open, and not the ones that are already closed (we can do this by looking at the `gov` module `EndBlocker` inside the Cosmos SDK.